### DISCUSSION TITLE
[Request]Add TranslatableWords like Jobs Reborn

### DISCUSSION THREAD

**Discussion by Hendrix-Shen** [Zrips/CMI#695] [Feature Requests]
**Description of issue or feature request:**
Add TranslatableWords can allow CMI to support display any minecraft blocks, mobs, enchanting, effect in any language

**Comment by CloudeLecaw** [comment]
uff you know this would be a big Change for the whole langfile? why not simply let it Show in your Client Version Name no matter what lang the Server has….
